WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -611,7 +611,9 @@ extension OrderDetailsViewModel {
                 switch result {
                 case .success(let subscriptions):
                     self?.dataSource.orderSubscriptions = subscriptions
-                    // TODO: 9238 - Add analytics
+                    if subscriptions.isNotEmpty {
+                        ServiceLocator.analytics.track(event: .Orders.subscriptionsShown())
```

Comment:
Wouldn't we get multiple events if the user refreshes the view? Or is that expected?

Reply:
That's expected. Since we don't know if there are subscriptions to display for an order until they are synced, we expect to track this each time an order is loaded/refreshed. (You can see the [equivalent tracking on Android](https://github.com/woocommerce/woocommerce-android/blob/24fbc02827e09c166c5f6ef5c79b5985bf4ebea4/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/details/OrderDetailViewModel.kt#L664-L669) for comparison.)